src/setup/devnet/templates/docker/docker-bor-start.sh.njk

```diff
@@ -2,25 +2,19 @@
 
 set -x
 
-NODE_DIR=/root/.bor
-DATA_DIR=/root/.bor/data
-
```

Comment:
Was there any reason why we didn't define `/var/lib/bor` as a constant and use it below in the script ?
Also I think you'll have to make the changes here in [docker-compose](https://github.com/maticnetwork/matic-cli/blob/977270c9d411278e4c3603b01aa2287bca2a4537/src/setup/devnet/templates/docker/docker-compose.yml.njk#L77C3-L79) to mount the volumes corectly. 
For example, see [this](https://github.com/maticnetwork/matic-cli/blob/977270c9d411278e4c3603b01aa2287bca2a4537/src/setup/devnet/templates/docker/docker-compose.yml.njk#L46-L48) for heimdall

Reply:
Because we are anyway hardcoding it in the `config.toml` file, I did the same here as well.

Reply:
But in script I think we can declare it as a constant.

Reply:
Done, [here](https://github.com/maticnetwork/matic-cli/pull/215/commits/80a4181e29019ffe3b0d5454b56f13689cff4f33).